public/javascripts/SVValidate/src/mission/Mission.js

```diff
@@ -83,12 +85,19 @@ function Mission(params) {
      *                      progress will increase.
      */
     function updateMissionProgress(skip) {
+        // console.log(getProperties());
         let labelsProgress = getProperty("labelsProgress");
+        let missionsCompleted = getProperty("missionsCompleted")
         if (labelsProgress < getProperty("labelsValidated")) {
             if (!skip) {
                 labelsProgress += 1;
             }
-            svv.statusField.updateLabelCounts(labelsProgress);
+            var missionLabelNumber = labelsProgress + svv.validationsCompleted;
```

Comment:
This seems like kind of a messy way to keep track of the number of completed validations. What if instead of recomputing it every time, the statusField object just kept track of the count, and then when a label is validated, a function is called that increments that count?